### ISSUE TITLE
Fewest Moves appearing twice in competition results

### ISSUE THREAD

**Issue by pedrosino** [thewca/worldcubeassociation.org#2689]
See https://www.worldcubeassociation.org/competitions/SilentSwapOpen2017
![image](https://user-images.githubusercontent.com/1881933/37425136-450cf3c6-27ba-11e8-87d8-ebe85248a45c.png)

My guess is that it's due to people being tied as 1st place.
I think it would make more sense to show the event only once and both names under it.

**Comment by jfly** [comment]
This is a duplicate of https://github.com/thewca/worldcubeassociation.org/issues/1652, which you filed! You've gotten better at naming issues in the last year =)

**Comment by pedrosino** [comment]
Haha
I updated the title on that one.
